content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@davidjgonzalez -- I don't see why it would have to be part of the commit hook. As long as it updated in a reasonable time.